### ISSUE TITLE
When opening a terminal in vscode, it opens in HOME instead of the project dir

### ISSUE THREAD

**Issue by Butanium** [jkminder/dlab-runai-images#1]

**Comment by jkminder** [comment]
Yes this is due to a line in the .bashrc, that is copied to your HOME. I added it recently because without it you open '/' when connecting normally. But you're right it might be the wrong approach. It is easily fixable, by removing the line 
`cd ~` from your .bashrc (`/dlabscratch1/dlabscratch1/${GASPAR_USERNAME}/.bashrc`). 

Could you confirm, whether this fixes the issue?

**Comment by Butanium** [comment]
I guess the easiest fix would be to put a `if pwd==/ then cd ~`

**Comment by jkminder** [comment]
Good idea! https://github.com/jkminder/dlab-runai-images/commit/1ec663cf61dbc6fe8ef2ea17ed8609f7b2e4307e
